deployment/roles/nlx/defaults/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+# Kubernetes namespace
+nlx_namespace: nlx
+nlx_instance: proeftuin
+
+nlx_inway_version: latest
+nlx_inway_image: nlxio/inway:{{ nlx_inway_version }}
+
+# Service config - which services are reachable through NLX?
+nlx_inway_services: []
+# Example:
+# nlx_inway_services:
+#   - name: zrc
+#     endpoint_url: http://gemma-zrc.k8s.dc1.proeftuin.utrecht.nl/api/v1
+#     documentation_url = "http://gemma-zrc.k8s.dc1.proeftuin.utrecht.nl"
+#     authorization_model = "none"
+#     api_specification_document_url = "http://gemma-zrc.k8s.dc1.proeftuin.utrecht.nl/api/v1/schema/openapi.json"
+
+nlx_inway_replicas: 1
+nlx_inway_directory_registration_address: directory-registration-api.demo.nlx.io:443
+nlx_inway_self_address: inway.utrechtproeftuin.nl
```

Comment:
Should be inway.openzaak.nl?

Reply:
yeah these are defaults anyway, so I'll change it to a more generic case like `inway.gemeente.nl`